Library/Formula/wren.rb

```diff
@@ -0,0 +1,19 @@
+class Wren < Formula
+  homepage "http://munificent.github.io/wren/"
+  url "https://github.com/munificent/wren/archive/efd161cea278e5cc9d6b404451443cf48d4057a8.tar.gz"
```

Comment:
Yes, if there's no stable version we can't accept it in this repository - sorry! `homebrew/head-only` is a place for unversioned software.